### PR TITLE
fix some bugs, close #41

### LINKED ISSUE #41: bug: esbuild prompt "Could not resolve 'util'"
```
esbuild src/browser.ts --outfile=lib/browser.mjs --bundle --sourcemap --platform=neutral --external:cosmokit --external:supports-color --target=esnext
X [ERROR] Could not resolve "util"

    node_modules/object-inspect/util.inspect.js:1:25:
      1 │ module.exports = require('util').inspect;
        ╵                          ~~~~~~

  The package "util" wasn't found on the file system but is built into node. Are you trying to
  bundle for node? You can use "--platform=node" to do that, which will remove this error.
```

### PR DESCRIPTION
**Pull request by idanran**
Fixes #41 

Other:
feat: add dts
chore: upgrade dep

### CODE REVIEW

**Comment by idanran** [comment]
@ljharb PTAL

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+interface opts {
+    quoteStyle?: 'single' | 'double'
```
Comment (ljharb) [inline index.d.ts:2]: Types don’t belong in the package; they should only be in DefinitelyTyped.

File: index.js [hunk 2]
```diff
@@ -65,10 +65,6 @@ function addNumericSeparator(num, str) {
     return $replace.call(str, sepRegex, '$&_');
 }
 
-var utilInspect = require('./util.inspect');
```
Comment (ljharb) [inline index.js:68]: This is incorrect; the util module is always present, and the browser field handles that for working bundlers.

File: package.json [hunk 3]
```diff
@@ -3,14 +3,15 @@
   "version": "1.12.2",
   "description": "string representations of objects in node and the browser",
   "main": "index.js",
+  "typings": "index.d.ts",
```
Comment (ljharb) [inline package.json:6]: Please remove this.

**Review by ljharb — CHANGES_REQUESTED** [review]

**Comment by idanran** [comment]
Thanks for the review. I agree with your thoughts. This PR should be closed.

**Pull request closed without merging**